recipes-extend/host-readme/files/env/Readme.md

```diff
@@ -0,0 +1,16 @@
+# RZ/G2L-SBC env folder
+
+## Description
+
+This directory contains the environment file, which includes Yocto build environment variables.
+
+## A top-level directory of env folder
+
+```
+env                             
+├── core-image-minimal.env           <---- Environment file specific to core-image-minimal in yoctobuild
```

Comment:
You can make it more generic instead of targeting specific items (e.g., core-image-minimal, Qt, BSP, Weston, etc.), and also apply this approach to other README files.

Reply:
Updated in the commit: fa4a70cd5eb6037a6490416b3944b4b8ce0fae77